deepchem/models/torch_models/chemberta.py

```diff
@@ -0,0 +1,174 @@
+"""
+Huggingface/transformers RoBERTa model for sequence-based property prediction.
+"""
+
+from numpy.lib.npyio import save
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.utils.data import random_split
+import os
+
+class ChemBERTaMaskedLM(nn.Module):
+    # defaults here match HF model: DeepChem/SmilesTokenizer_PubChem_1M.
+    # TODO - figure out which to use as default, and which to pass to model class
+    def __init__(self,
+                vocab_size =600,
+                max_position_embedddings = 515,
```

Comment:
We'll want to add type annotations here for these arguments

Reply:
good catch! will do